rust/main/chains/hyperlane-aleo/src/config.rs

```diff
@@ -16,10 +16,16 @@ pub struct ConnectionConf {
     pub validator_announce_program: String,
     /// Chain Id
     pub chain_id: u16,
+    /// Proofing service
+    pub proofing_service: Vec<Url>,
```

Comment:
It looks like we are moving code base to use `proving` instead of `proofing`. Let's make it consistent across the code base.